### DISCUSSION TITLE
Multilanguage support

### DISCUSSION THREAD

**Discussion by k11q** [replit/kaboom#669] [Ideas]
Hi, I am Khairul from Malaysia. Thank you replit and team for making such a user friendly, fun and easy language to start programming.

I think kaboomjs is perfect for beginners and for people to start learning to program and code for the first time. I am thinking this language would be perfect for teaching kids to code for the first time, however since I am from Malaysia and our native language isn't English, I find the language barrier would add another layer of complexity for the kids, after all we want them to explore these by themselves.

I wonder if we can add multilanguage support for the docs, I would gladly put my time contributing for the Malay language. I hope you guys consider this and maybe I can get to writing as soon as possible, thank you!

**Comment by lajbel** [comment]
What do you mean with multiple language? Change from `body()` to `badan()` (I think that's `body` in Malay)?

**Reply by k11q** [reply]
sorry I didn't explain it clearly. I meant for the docs only.

**Reply by rBok6** [reply]
> sorry I didn't explain it clearly. I meant for the docs only.

Well Kaboom docs are open source, you could translate them yourself and later make a pull request to Kaboom's main branch.

**Comment by lajbel** [comment]
Well, the problem it's how docs works. In Kaboom.js source code, we have `src/types.ts`, and it has all descriptions of all functions, methods, interfaces, and other things. Make a translation of a file with more than 5000 lines won't be so viable. The best thing that I can recommend is use a translator extension to translate all the site, like [Google Translate](https://chrome.google.com/webstore/detail/google-translate/aapbdbdomjkkjkaonfhkkikfgjllcleb?hl=es-419). This will translate all description, but not the code.

![image](https://user-images.githubusercontent.com/71136486/220788199-749107aa-7b6c-4832-8ff9-8d91024870ef.png)
